app/controllers/state_file/questions/verification_code_controller.rb

```diff
@@ -48,10 +48,19 @@ def update
       private
 
       def get_existing_intake(intake)
-        search = intake.class.where.not(id: intake.id, raw_direct_file_data: nil)
-        search = search.where(phone_number: intake.phone_number) if intake.phone_number.present?
-        search = search.where(email_address: intake.email_address) if intake.email_address.present?
-        search.first
+
+        state_intake_classes = StateFile::StateInformationService.state_intake_classes
+
+        state_intake_classes.each do |intake_class|
+          search = intake_class.where.not(id: intake.id, raw_direct_file_data: nil)
+          search = search.where(phone_number: intake.phone_number) if intake.phone_number.present?
+          search = search.where(email_address: intake.email_address) if intake.email_address.present?
```

Comment:
what happens if `intake.phone_number` & `intake.email_address` is not present?

Reply:
this only runs if the form is valid which requires one of those to be present.